core/lib/wallet_creator/src/main.rs

```diff
@@ -0,0 +1,70 @@
+use std::{env};
+
+use zksync::{
+    web3::{
+        types::{H160, H256},
+    },
+    zksync_types::{
+        tx::PackedEthSignature,
+    },
+    Network, RpcProvider, Wallet, WalletCredentials,
+};
+
+use zksync_eth_signer::{PrivateKeySigner};
+
+fn eth_random_account_credentials() -> (H160, H256) {
+    let mut eth_private_key = H256::default();
+    eth_private_key.randomize();
+
+    let address_from_pk = PackedEthSignature::address_from_private_key(&eth_private_key).unwrap();
+
+    (address_from_pk, eth_private_key)
+}
+
+async fn create_new_wallet(address: &str, key: &str) -> Result<Wallet<PrivateKeySigner, RpcProvider>, anyhow::Error>{
+    let mut eth_address:H160;
+    let mut eth_private_key:H256;
+    if(address.is_empty() || key.is_empty()) {
+        (eth_address, eth_private_key) = eth_random_account_credentials();
+    } else {
+        eth_address = ETH_ADDR.parse().unwrap();
+        eth_private_key = ETH_PRIVATE_KEY.parse().unwrap();
+    }
+
+    println!("-> Address {:?}", eth_address);
+    println!("-> Private key {:?}", eth_private_key);
+
+    let eth_signer = PrivateKeySigner::new(eth_private_key);
+    let credentials =
+        WalletCredentials::from_eth_signer(eth_address, eth_signer, Network::Localhost)
+            .await
+            .unwrap();
+
+    let provider = RpcProvider::new(Network::Localhost);
+    let wallet = Wallet::new(provider, credentials).await?;
+
+
+    Ok(wallet)
+}
+
+#[tokio::main]
+async fn main() -> Result<(), anyhow::Error> {
+    let args: Vec<String> = env::args().collect();
+
+    let mut address: &str = "";
+    let mut key: &str = "";
+
+    if(args.len() > 3){
```

Comment:
I get this suggestion if I build it locally
```suggestion
    if args.len() > 3 {
```

Reply:
Updated